packages/kit/src/runtime/server/page/respond.js

```diff
@@ -185,11 +185,11 @@ export async function respond(opts) {
 				}
 			}
 
-			if (loaded && loaded.loaded.context) {
+			if (loaded && loaded.loaded.stuff) {
 				// TODO come up with better names for stuff
```

Comment:
Can we get rid of this TODO finally? :)

Were there other things that we still wanted to find better names for pre-1.0?

Reply:
Yeah, I think we should get rid of the TODO :smile: